objgraph.py

```diff
@@ -718,25 +734,28 @@ def show_graph(objs, edge_func, swap_source_target,
         dot.wait()
         if dot.returncode != 0:
             # XXX: shouldn't this go to stderr or a log?
-            print("dot failed to generate '%s' image: output format not supported?")
+            print('dot failed to generate "%s" image:',
+                  'output format not supported?')
         f.close()
         print("Image generated as %s" % filename)
     else:
         if filename:
-            print("Graph viewer (xdot) and image renderer (dot) not found, not doing anything else")
+            print("Graph viewer (xdot) and image renderer (dot) not found,"
+                  "not doing anything else")
```

Comment:
I have a rule: when splitting a string over multiple lines by using concatenation, always make the continuation lines start with a space.

That makes it easy to notice bugs like you just introduced here: missing space after comma in "... not found,not doing anything else".